gamestonk_terminal/helper_funcs.py

```diff
@@ -106,24 +113,45 @@ def plot_view_stock(df: pd.DataFrame, symbol: str, interval: str):
     # In order to make nice Volume plot, make the bar width = interval
     if interval == "1440min":
         bar_width = timedelta(days=1)
+        title_string = "Daily"
     else:
         bar_width = timedelta(minutes=int(interval.split("m")[0]))
+        title_string = f"{int(interval.split('m')[0])} min"
 
-    plt.bar(df.index, df.iloc[:, -1], color="k", alpha=0.8, width=bar_width)
-    plt.ylabel("Volume")
-    _ = axVolume.twinx()
-    plt.plot(df.index, df.iloc[:, :-1])
-    plt.title(symbol.upper() + " (Time Series)")
-    plt.xlim(df.index[0], df.index[-1])
-    plt.xlabel("Time")
-    plt.ylabel("Share Price ($)")
-    plt.legend(df.columns)
-    plt.grid(b=True, which="major", color="#666666", linestyle="-")
-    plt.minorticks_on()
-    plt.grid(b=True, which="minor", color="#999999", linestyle="-", alpha=0.2)
-
+    ax[0].yaxis.tick_right()
+    if "Adj Close" in df.columns:
+        ax[0].plot(df.index, df["Adj Close"], c=cfgPlot.VIEW_COLOR)
+    else:
+        ax[0].plot(df.index, df["Close"], c=cfgPlot.VIEW_COLOR)
+    ax[0].set_xlim(df.index[0], df.index[-1])
+    ax[0].set_xticks([])
+    ax[0].yaxis.set_label_position("right")
+    ax[0].set_ylabel("Share Price ($)")
+    ax[0].grid(axis="y", color="gainsboro", linestyle="-", linewidth=0.5)
+
+    ax[0].spines["top"].set_visible(False)
+    ax[0].spines["left"].set_visible(False)
+    ax[1].bar(df.index, df.Volume, color=bar_colors, alpha=0.8, width=bar_width)
```

Comment:
What about making df.Volume/1e6 and putting a label in Million. 

Otherwise it gets that 1e8 number on top which is quite ugly.

Since you're making our "view" pretty, might as well make it top notch eheh